server/src/main/java/io/crate/analyze/TableIdentsExtractor.java

```diff
@@ -145,6 +146,11 @@ public Collection<RelationName> visitParameterSymbol(ParameterSymbol parameterSy
         public Collection<RelationName> visitSelectSymbol(SelectSymbol selectSymbol, Void context) {
             return selectSymbol.relation().accept(RELATION_TABLE_IDENT_EXTRACTOR, context);
         }
+
+        @Override
+        public Collection<RelationName> visitAlias(AliasSymbol aliasSymbol, Void context) {
```

Comment:
Would it make sense to use `DefaultTraversalSymbolVisitor` for this visitor? Then this override afaik wouldn't be necessary.

Reply:
Thanks made the change but it required some more changes.